app/presenters/tree_builder_chargeback_reports.rb

```diff
@@ -40,6 +40,6 @@ def x_get_tree_custom_kids(object, count_only, _options)
                              .select(:id, :miq_report_id, :name, :last_run_on, :miq_task_id)
                              .order(:last_run_on => :desc)
 
-    count_only_or_objects(count_only, objects, "name")
+    count_only_or_objects(count_only, objects)
```

Comment:
I don't get this part, why are you dropping the `name` here? And if you're dropping it, why is the change above necessary?

Reply:
@skateman we are already sorting by `name` (in association) and `order(:last_run_on => :desc)` - no reason to add this sorting as well.